library/Fission/Types.hs

```diff
@@ -204,12 +204,12 @@ instance MonadRoute53 Fission where
 
 instance MonadDNSLink Fission where
   set _userId url@URL {..} zoneID (IPFS.CID hash) = do
-    Route53.set Cname url zoneID (pure $ textDisplay gateway) >>= \case
+    Route53.set Cname url zoneID (pure $ textDisplay gateway) 86400 >>= \case
       Left err ->
         return $ Error.openLeft err
 
       Right _ ->
-        Route53.set Txt dnsLinkURL zoneID (pure dnsLink) <&> \case
+        Route53.set Txt dnsLinkURL zoneID (pure dnsLink) 10 <&> \case
```

Comment:
👍 Good idea

Reply:
Not for right now, but down the line, we may want to make this user configurable, too. Like "on production, set TTL to an hour" or something.